MvvmCross/Platform/iOS/Views/MvxTabBarViewController.cs

```diff
@@ -77,6 +78,12 @@ protected virtual void SetTitleAndTabBarItem(UIViewController viewController, Mv
 
         public virtual bool ShowChildView(UIViewController viewController)
         {
+            if (MoreNavigationController?.ViewControllers?.Any() ?? false) 
```

Comment:
Suppose you have 6 tabs, all of them contain NavigationViewControllers. If you navigate to a child ViewController when the first Tab is selected  - you are not looking at MoreNavigationController at that time -. What would happen? Because it seems to me like this line would cause the child controller to be shown in the incorrect navigation stack.

Reply:
You are right.. Thought that MoreNavigationController is empty when you are not on Menu tab - looked at the app, it indeed broke child navigation. Working on fix..